Automattic/wp-calypso
# BBE Landing: Add localized images for slideshow

#### Proposed Changes

P2: pdh1Xd-1DL-p2

This PR updates the images used on the BBE landing page. Changes:
* Show a separate set of localized images if the locale is `es`.
* The earlier implementation had hardcoded values for the animation. This change refactors the component so that the animation is easier to understand, and any future changes to the images would not involve any changes to the animation.
* The number of files changed in this PR is large, so it might be easier to review the change one commit at a time.
* The files `theme-grid.png, vertical-creative.png, vertical-restaurant.png and vertical-services.png` are not being used, so I deleted those too.

#### Testing Instructions


* Switch to EN locale.
* Go to `/setup/site-setup/goals?siteSlug=<site slug>`.
* Confirm that the slideshow displays and animates correctly. You can verify that the animation is correct by comparing it to the live version (`https://wordpress.com/setup/site-setup/difmStartingPoint?siteSlug=<site slug>`)
* Switch to the ES locale and repeat the above steps.
* Confirm that localized images are shown.

#### Pre-merge Checklist





Fixes Automattic/martech#1329
